braindecode/datasets/base.py

```diff
@@ -290,3 +338,26 @@ def save(self, path, overwrite=False):
         if concat_of_raws:
             json.dump({'target_name': target_name}, open(target_file_name, 'w'))
         self.description.to_json(description_file_name)
+
+    @property
+    def description(self):
+        df = pd.DataFrame([ds.description for ds in self.datasets])
+        df.reset_index(inplace=True, drop=True)
+        return df
```

Comment:
Is this normally called during a training loop? If so, we should probably check if we get slowdowns from it. If not called during training loop, should not be a problem

Reply:
I assume it is not called?

Reply:
No, not called during training loop.